release/build/main.go

```diff
@@ -172,38 +180,9 @@ func hashreleaseSubCommands(cfg *config.Config, runner *registry.DockerRunner) [
 				// For real releases, release notes are generated prior to building the release. For hash releases,
 				// generate a set of release notes and add them to the hashrelease directory.
 				tasks.ReleaseNotes(cfg, dir, version.New(ver))
-				return nil
-			},
-			After: func(c *cli.Context) error {
-				// We use an After() function to modify the generated release output to match
-				// the "legacy" format our CI tooling expects. This should be temporary until
-				// we can update the tooling to expect the new format.
-				// Specifically, we need to do two things:
-				// - Copy the windows zip file to files/windows/calico-windows-<ver>.zip
-				// - Copy tigera-operator-<ver>.tgz to tigera-operator.tgz
-				logrus.Info("Modifying hashrelease output to match legacy format")
-				pinned, err := hashrelease.RetrievePinnedVersion(cfg.TmpFolderPath())
-				if err != nil {
-					return err
-				}
-				ver := pinned.Components["calico"].Version
 
-				// Copy the windows zip file to files/windows/calico-windows-<ver>.zip
-				if err := os.MkdirAll(filepath.Join(dir, "files", "windows"), 0o755); err != nil {
-					return err
-				}
-				windowsZip := filepath.Join(dir, fmt.Sprintf("calico-windows-%s.zip", ver))
-				windowsZipDst := filepath.Join(dir, "files", "windows", fmt.Sprintf("calico-windows-%s.zip", ver))
-				if err := utils.CopyFile(windowsZip, windowsZipDst); err != nil {
-					return err
-				}
-
-				// Copy the operator tarball to tigera-operator.tgz
-				operatorTarball := filepath.Join(dir, fmt.Sprintf("tigera-operator-%s.tgz", ver))
-				operatorTarballDst := filepath.Join(dir, "tigera-operator.tgz")
-				if err := utils.CopyFile(operatorTarball, operatorTarballDst); err != nil {
-					return err
-				}
+				// Adjsut the formatting of the generated outputs to match the legacy hashrelease format.
+				tasks.ReformatHashrelease(cfg, dir)
```

Comment:
probably skip this for dev build since this is a legacy support.

Reply:
I'm not sure about that - for one, I think I'd need to add another flag to configure that. Also, if I am building this as a developer I want to make sure that what I am running is as close to the "real" code as possible - what if my changes break this logic?